processor/k8sprocessor/options.go

```diff
@@ -68,18 +70,20 @@ func WithExtractMetadata(fields ...string) Option {
 	return func(p *kubernetesprocessor) error {
 		if len(fields) == 0 {
 			fields = []string{
-				metdataNamespace,
+				metadataNamespace,
 				metadataPodName,
 				metadataPodUID,
 				metadataStartTime,
 				metadataDeployment,
 				metadataCluster,
 				metadataNode,
+				metadataHostName,
+				metadataContainerName,
```

Comment:
We could keep the values sorted here (and pretty much anytime there's a list)

Reply:
done